vulture/whitelists/argparse_whitelist.py

```diff
@@ -0,0 +1,7 @@
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.epilog
```

Comment:
Let's rename `parser` to `_`. Otherwise, Vulture will miss unused variables called `parser`.

Reply:
Oh yes, nice catch!